components/query_filter/utils.h

```diff
@@ -3,13 +3,17 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-#ifndef BRAVE_BROWSER_NET_BRAVE_QUERY_FILTER_H_
-#define BRAVE_BROWSER_NET_BRAVE_QUERY_FILTER_H_
+#ifndef BRAVE_COMPONENTS_QUERY_FILTER_UTILS_H_
+#define BRAVE_COMPONENTS_QUERY_FILTER_UTILS_H_
 
+#include <string>
 #include "third_party/abseil-cpp/absl/types/optional.h"
 
 class GURL;
 
 absl::optional<GURL> ApplyQueryFilter(const GURL& original_url);
+absl::optional<GURL> ApplyPotentialQueryStringFilter(const GURL& source,
```

Comment:
nit: source -> source_url destination -> destination_url for consistency

Reply:
done